assets/dev/js/editor/document/elements/commands-internal/set-settings.js

```diff
@@ -25,6 +25,13 @@ export class SetSettings extends $e.modules.editor.CommandContainerInternalBase
 				container.render();
 			}
 		} );
+
+		$e.store.dispatch(
```

Comment:
Go over all elements commands and move the Marionette to a separate method:
`updateMarionetteStore`